library/src/containers/AsyncApi/AsyncApi.tsx

```diff
@@ -182,18 +168,18 @@ class AsyncApiComponent extends Component<AsyncApiProps, AsyncAPIState> {
 
     const parsed = await this.parser.parse(schema, parserOptions);
     this.setState({
-      validatedSchema: this.beautifySchema(parsed.data),
+      validatedSchema: parsed.data, // this.beautifySchema(parsed.data),
       asyncapi: parsed.asyncapi,
       error: parsed.error,
     });
   }
 
-  private beautifySchema(schema: NullableAsyncApi): NullableAsyncApi {
-    if (!schema) {
-      return null;
-    }
-    return beautifier.beautify(schema);
-  }
+  // private beautifySchema(schema: NullableAsyncApi): NullableAsyncApi {
+  //   if (!schema) {
+  //     return null;
+  //   }
+  //   return beautifier.beautify(schema);
+  // }
```

Comment:
You probably can remove this.

Reply:
I commented it on the first com,it and forgot about it 😅  thanks!